pkg/crosscluster/logical/create_logical_replication_stmt.go

```diff
@@ -250,8 +255,23 @@ func createLogicalReplicationStreamPlanHook(
 			defaultConflictResolution = *cr
 		}
 
+		jobID := p.ExecCfg().JobRegistry.MakeJobID()
+		var reverseStreamCmd string
+		if stmt.CreateTable && options.BidirectionalURI() != "" {
+			// TODO: validate URI.
```

Comment:
Validating the URI is a bit tricky. We really need the peer cluster to validate it.

Reply:
>We really need the peer cluster to validate it.

Good point. I can plan to pass it as an extra arg to `CreateForTables`, so the og source can validate it. Sounds reasonable?